dashboard/src/features/shared/components/ErrorBanner/ErrorBanner.jsx

```diff
@@ -3,10 +3,25 @@ import styles from './ErrorBanner.scss'
 
 class ErrorBanner extends React.Component {
   render() {
+    let error = this.props.error || {}
+    if (typeof error == 'string') {
+      error = {chainMessage: error}
+    }
+
     return (
       <div className={styles.main}>
         {this.props.title && <strong>{this.props.title}<br/></strong>}
-        {this.props.message}
+
+        {error.chainMessage &&
```

Comment:
Is there a fallback here in case `chainMessage` is missing, i.e. in the case of other JavaScript errors that are not generated by the client SDK?